angular-frontend/_app/templates/people.template.html

```diff
@@ -1,51 +1,71 @@
-            padding: 10px;
-            padding-left: 25px;
-        }
-    </style>
-    <body>
-        <div class="container" id="form">
-            <md-content>
-                <h2>Add a Person</h2>
-                <div>
-                    <md-input-container class="md-input-had-value">
-                        <label for="first">First Name</label>
-                        <input class="ng-pristine ng-valid md-input ng-not-empty ng-touched" id="first" style/>
-                    </md-input-container>
-                    <md-input-container class="md-input-had-value">
-                        <label for="middle">Middle Name</label>
-                        <input class="ng-pristine ng-valid md-input ng-not-empty ng-touched" id="middle" style/>
-                    </md-input-container>
-                    <md-input-container class="md-input-had-value">
-                        <label for="first">Last Name</label>
-                        <input class="ng-pristine ng-valid md-input ng-not-empty ng-touched" id="last" style/>
-                    </md-input-container>
-                    <md-input-container class="_md-datepicker-floating-label _md-datepicker-has-calendar-icon">
-                        <label for="birthdate">Date of Birth</label>
-                        <md-datepicker ng-model="birthDate" id="birthdate"></md-datepicker>
-                    </md-input-container>
-                    <button class="md-button md-ink-ripple" type="submit">
-                        <span>SUBMIT</span>
-                    </button>
-                </div>
-            </md-content>
-        </div>
-        <hr>
-        <div class="container" id="form">
-            <!-- This section should show most recent people added, and offer the ability -->
-            <!-- to add a relation for the new family member to other people in the list -->
-            <h2>Your People</h2>
-            <table></table>
-        </div>
-    </body>
-</div>
-
 <style>
+.form {
+    padding: 10px;
+    padding-left: 25px;
+}
+
 .container {
     margin: auto;
     width: 50%;
     padding: 10px;
 }
+
+#saveRelation {
+    padding-top: 10px;
+}
+
+#savePeople {
+    text-align: center;
+    margin: auto;
+    width: 50%;
+    padding: 10px;
+}
+
+#nameField {
+    width: 40%;
+    border-radius: 4px;
+}
 </style>
+  
+<body>
+    <div class="form" id="savePeople">
+        Add a person to your tree:
+        <form id="saveRelation">
+            <!-- On save, send to db -->
+            Name: <input type="text" name="firstname" id='nameField'> <input type="submit" value="Save">
+        </form>
+    </div>
+    <div class="container" id="form">
+        <md-content>
+            <h2>Add a Person</h2>
+            <div>
+                <md-input-container class="md-input-had-value">
+                    <label for="first">First Name</label>
+                    <input class="ng-pristine ng-valid md-input ng-not-empty ng-touched" id="first" style/>
```

Comment:
These classes like 'ng-pristinne' and 'ng-valid' should be maintained by the Angular application rather than being hardcoded into the application.